CHANGELOG.md

```diff
@@ -1,3 +1,234 @@
+## [0.22.0-dev.8](https://github.com/dashevo/platform/compare/v0.21.8...v0.22.0-dev.8) (2022-03-01)
+
+
+### ⚠ BREAKING CHANGES
+
+* **dpp:** do not allow to index array properties (#225)
```

Comment:
I think you're autogenerating the changelog, but occasionally the referenced PR isn't actually linked to like in this case.

Reply:
Make sense @shuplenkov